net/gatling/Makefile

```diff
@@ -0,0 +1,45 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gatling
+PKG_VERSION:=0.16
+PKG_RELEASE:=1
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/gatling-$(PKG_VERSION)
+PKG_SOURCE:=gatling-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://www.fefe.de/gatling/gatling-0.16.tar.xz
+PKG_HASH:=5f96438ee201d7f1f6c2e0849ff273b196bdc7493f29a719ce8ed08c8be6365b
+
+PKG_MAINTAINER:=Martin Hübner <martin.hubner@web.de>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/gatling
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Web Servers/Proxies
+	TITLE:=gatling - a high performance web server
+	URL:=https://www.fefe.de/gatling/
+	PKG_BUILD_DEPENDS:=openssl px5g-mbedtls libowfat
+	DEPENDS:=+zlib +libcap +libopenssl
+#	DEPENDS:=+zlib +mbedtls +libcap +libopenssl
+endef
+
+define Package/gatling/description
+	gatling - a high performance web server
+	with a a rather small binary size but a decent set of extra features
+	like ftp, basic htaccess and basic DoS-Protection.
+	Gatling is particularly good in situations with very high load.
+endef
+
+define Build/Configure
+	$(call Build/Configure/Default)
+endef
```

Comment:
Why?

Reply:
It was in the example makefile in the Wiki. Is that unnecessary too?

Reply:
This is not necessary and is done Implicity

Reply:
I removed it now.